pkg/auth/dependency/oidc/id_token.go

```diff
@@ -35,10 +35,13 @@ func (ti *IDTokenIssuer) IssueIDToken(client config.OAuthClientConfiguration, us
 		Nonce: nonce,
 	}
 
-	key, err := jwt.ParseRSAPrivateKeyFromPEM([]byte(ti.OIDCConfig.Keys[0].PrivateKey))
+	key := ti.OIDCConfig.Keys[0]
```

Comment:
Check the length of `Keys` first to avoid panic.

Reply:
I think it's better to enforce non-emptiness at config schema level? Otherwise, the whole OIDC flow cannot work.